In file included from layers/state_tracker.cpp:28:
layers/generated/vk_enum_string_helper.h:5559:14: error: use of undeclared identifier 'VK_VIDEO_CODING_QUALITY_PRESET_DEFAULT_BIT_KHR'
        case VK_VIDEO_CODING_QUALITY_PRESET_DEFAULT_BIT_KHR:
             ^

--- layers/generated/parameter_validation.cpp.orig	2021-08-30 21:04:49 UTC
+++ layers/generated/parameter_validation.cpp
@@ -80,7 +80,7 @@ const VkDisplayPlaneAlphaFlagsKHR AllVkDisplayPlaneAlp
 const VkVideoChromaSubsamplingFlagsKHR AllVkVideoChromaSubsamplingFlagBitsKHR = VK_VIDEO_CHROMA_SUBSAMPLING_INVALID_BIT_KHR|VK_VIDEO_CHROMA_SUBSAMPLING_MONOCHROME_BIT_KHR|VK_VIDEO_CHROMA_SUBSAMPLING_420_BIT_KHR|VK_VIDEO_CHROMA_SUBSAMPLING_422_BIT_KHR|VK_VIDEO_CHROMA_SUBSAMPLING_444_BIT_KHR;
 const VkVideoCodecOperationFlagsKHR AllVkVideoCodecOperationFlagBitsKHR = VK_VIDEO_CODEC_OPERATION_INVALID_BIT_KHR|VK_VIDEO_CODEC_OPERATION_ENCODE_H264_BIT_EXT|VK_VIDEO_CODEC_OPERATION_DECODE_H264_BIT_EXT|VK_VIDEO_CODEC_OPERATION_DECODE_H265_BIT_EXT;
 const VkVideoCodingControlFlagsKHR AllVkVideoCodingControlFlagBitsKHR = VK_VIDEO_CODING_CONTROL_DEFAULT_KHR|VK_VIDEO_CODING_CONTROL_RESET_BIT_KHR;
-const VkVideoCodingQualityPresetFlagsKHR AllVkVideoCodingQualityPresetFlagBitsKHR = VK_VIDEO_CODING_QUALITY_PRESET_DEFAULT_BIT_KHR|VK_VIDEO_CODING_QUALITY_PRESET_NORMAL_BIT_KHR|VK_VIDEO_CODING_QUALITY_PRESET_POWER_BIT_KHR|VK_VIDEO_CODING_QUALITY_PRESET_QUALITY_BIT_KHR;
+const VkVideoCodingQualityPresetFlagsKHR AllVkVideoCodingQualityPresetFlagBitsKHR = VK_VIDEO_CODING_QUALITY_PRESET_NORMAL_BIT_KHR|VK_VIDEO_CODING_QUALITY_PRESET_POWER_BIT_KHR|VK_VIDEO_CODING_QUALITY_PRESET_QUALITY_BIT_KHR;
 const VkVideoComponentBitDepthFlagsKHR AllVkVideoComponentBitDepthFlagBitsKHR = VK_VIDEO_COMPONENT_BIT_DEPTH_INVALID_KHR|VK_VIDEO_COMPONENT_BIT_DEPTH_8_BIT_KHR|VK_VIDEO_COMPONENT_BIT_DEPTH_10_BIT_KHR|VK_VIDEO_COMPONENT_BIT_DEPTH_12_BIT_KHR;
 const VkVideoSessionCreateFlagsKHR AllVkVideoSessionCreateFlagBitsKHR = VK_VIDEO_SESSION_CREATE_DEFAULT_KHR|VK_VIDEO_SESSION_CREATE_PROTECTED_CONTENT_BIT_KHR;
 const VkVideoDecodeFlagsKHR AllVkVideoDecodeFlagBitsKHR = VK_VIDEO_DECODE_DEFAULT_KHR|VK_VIDEO_DECODE_RESERVED_0_BIT_KHR;
--- layers/generated/vk_enum_string_helper.h.orig	2021-08-30 21:04:49 UTC
+++ layers/generated/vk_enum_string_helper.h
@@ -5556,8 +5556,6 @@ static inline const char* string_VkVideoCodingQualityP
 {
     switch (input_value)
     {
-        case VK_VIDEO_CODING_QUALITY_PRESET_DEFAULT_BIT_KHR:
-            return "VK_VIDEO_CODING_QUALITY_PRESET_DEFAULT_BIT_KHR";
         case VK_VIDEO_CODING_QUALITY_PRESET_NORMAL_BIT_KHR:
             return "VK_VIDEO_CODING_QUALITY_PRESET_NORMAL_BIT_KHR";
         case VK_VIDEO_CODING_QUALITY_PRESET_POWER_BIT_KHR:
